docs/pages/distribution/runtime-version.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Runtime Version
+---
+
+Expo Update allows you to publish over the air updates to apps. The update must be for an app running a
+[compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. You may use a runtime other than an Expo SDK, by specifying a runtime version.
+
+The runtime version should be specified in your `app.json`:
+
+```typescript
+{
+	expo: {
+		...
+		runtimeVersion: "2.718",
+	}
+}
+```
+## Publishing
+
+[Publishes](../workflow/publishing/#how-to-publish) run with the runtime version set in the `app.json` will be delivered to builds running the same runtime version.
+
+## Builds
+
+Apps running a custom runtime version can be built with [EAS Build](../build/introduction/).
+
+There are two ways to set the runtime version of a build.
+
+1. (Recommended) After setting the runtime version in your `app.json`, run `expo prebuild`.
+2. Manually edit the Plist/AndroidManifest.xml:
+
+	### iOS
+
+	```diff
+	+ <key>EXUpdatesRuntimeVersion</key>
+    + <string>2.718</string>
```

Comment:
indentation looks off?

Reply:
This might be glitchy behavior with GHM since the preview looks OK but it would be nice for the source to look well-formatted if possible.

Reply:
Oh weird, this is. a tab/space issue that rendered as equal on my local setup. Fixed.